### PR TITLE
[wdspec] add a test for contextCreated events on subcribe

### PR DESCRIPTION
**Pull request by OrKoN**
Firefox impl tracking issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1754273
Chrome impl tracking issue: https://github.com/GoogleChromeLabs/chromium-bidi/issues/1902

### CODE REVIEW

File: webdriver/tests/bidi/browsing_context/context_created/context_created.py [hunk 1]
```diff
@@ -263,3 +263,22 @@ async def on_event(method, data):
...
+    context_info = await wait_for_future_safe(on_entry)
+
+    assert_browsing_context(
+        context_info,
+        top_level_context["context"],
+        children=None,
+        url="about:blank",
+        parent=None,
+        user_context="default"
+    )
```
Comment (juliandescottes) [inline webdriver/tests/bidi/browsing_context/context_created/context_created.py:284]: We haven't implemented this in Firefox yet, so I can't test it. But at this point we should have at least 2 contexts: one for the usual top_context, and one for the context created in the test.

Meaning we should at least get 2 events, and I'm not sure the spec is clear about which event should come first. It sounds fragile to assume the first event will be for `top_level_context`?

I imagine we can leverage https://github.com/web-platform-tests/wpt/issues/44702 once it lands. 
In the meantime, maybe just collect events in an array, wait for 0.5 seconds and check that we have at least one event for top_level_context?

Comment (juliandescottes) [reply webdriver/tests/bidi/browsing_context/context_created/context_created.py:284]: Oh right I didn't notice that! Sorry for the noise.

**Review by juliandescottes — APPROVED** [review]
LGTM. For the record implementation in Firefox is tracked at https://bugzilla.mozilla.org/show_bug.cgi?id=1754273

File: webdriver/tests/bidi/browsing_context/context_created/context_created.py (hunk 1 above)
Comment (OrKoN) [reply webdriver/tests/bidi/browsing_context/context_created/context_created.py:284]: Ah, no, you were totally correct. I have added it address your comment.

Comment (whimboo) [reply webdriver/tests/bidi/browsing_context/context_created/context_created.py:284]: Just a note that we might want to have a test for each a new tab, window, and (nested?) frame at the same time. But maybe we can do that once the PR @juliandescottes mentioned landed?

**Pull request merged**